TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -23,7 +23,7 @@
 @TeleOp(name = "main")
 public class Robot extends LinearOpMode {
 
-    ElapsedTime time = new ElapsedTime();
+    ElapsedTime robotTime = new ElapsedTime();
```

Comment:
Why did you change this variable's name?

Reply:
it is a more acuurate name, dont you think so?

Reply:
No.